content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst

```diff
@@ -1,137 +1,224 @@
+by default, with the full :guilabel:`Reserved` quantity taken from that specific lot (if there is
+enough stock in that particular lot). If there is insufficient stock in that lot, or if partial
+quantities of the :guilabel:`Demand` should be taken from multiple lots, change the quantity in the
+:guilabel:`Done` column to only include part of the total quantity.
+
+.. note::
+    The lot automatically chosen for delivery orders will vary depending on the selected removal
+    strategy (:abbr:`FIFO (First In, First Out)`, :abbr:`LIFO (Last In, First Out)`, or
+    :abbr:`FEFO (First Expiry, First Out)`). It will also depend on the quantity ordered, and if
+    there is enough quantity in one lot to fulfill the order.
+
+.. seealso::
+    - :doc:`../../routes/strategies/removal`
+
+Then, click :guilabel:`Add a line`, select an additional (different) :guilabel:`Lot/Serial Number`,
+apply the rest of the :guilabel:`Done` quantities, and click :guilabel:`Confirm` to close the
+popup. Click :guilabel:`Validate` to deliver the products.
+
+.. image:: lots/lots-detailed-operations-popup.png
+   :align: center
+   :alt: Detailed operations popup for source lot number on sales order.
+
+A :guilabel:`Traceability` smart button will appear upon validating the delivery order. Click the
+:guilabel:`Traceability` smart button to see the updated :guilabel:`Traceability Report`, which
+includes: a :guilabel:`Reference` document; the :guilabel:`Product` being traced; the
+:guilabel:`Lot/Serial #` assigned; and the :guilabel:`Reference` receipt from the previous purchase
+order (if the product quantities shared the same lot number).
+
+Manage lots for different operations types
+==========================================
+
+By default in Odoo, the creation of new lots is only allowed upon *receiving* products from a
+purchase order. *Existing* lot numbers cannot be used. For sales orders, the opposite is true: new
+lot numbers cannot be created on the delivery order, only existing lot numbers can be used.
+
+To change the ability to use new or existing lot numbers on any operation type, go to
+:menuselection:`Inventory --> Configuration --> Operations Types`, and select the desired
+:guilabel:`Operation Type`. For :guilabel:`Receipts`, the
+:guilabel:`Use Existing Lots/Serial Numbers` option can be enabled by clicking :guilabel:`Edit` and
+then clicking the :guilabel:`checkbox`. For :guilabel:`Delivery Orders`, the
+:guilabel:`Create New Lots/Serial Numbers` option can be enabled by clicking :guilabel:`Edit` and
+clicking the checkbox. For any changes made, be sure to :guilabel:`Save`.
+
+.. image:: lots/lots-operations-type-form.png
+   :align: center
+   :alt: Enabled traceability setting on operations type form.
+
+.. tip::
+    For inter-warehouse trasnfers involving products tracked by lots, it can be useful to enable
+    the :guilabel:`Use Existing Lots/Serial Numbers` option for :guilabel:`Receipts`.
```

Comment:
would a :guilabel: be necessary for Receipts in this instance? Might be worth looking into...